pages/common/pio-package.md

```diff
@@ -0,0 +1,37 @@
+# pio package
+
+> Manage packages in the registry.
+> Packages can only be removed within 72 hours (3 days) from the date that they are published.
+> More information: <https://docs.platformio.org/en/latest/core/userguide/package/>.
+
+- Create a package tarball from the current directory:
+
+`pio package pack --output {{path/to/output_tarball.tar.gz}}`
```

Comment:
The command argument is named `--output` and `tarball` doesn't add a meaning to the output file. I think `package.tar.gz` or maybe `my_package.tar.gz`
```suggestion
`pio package pack --output {{path/to/package.tar.gz}}`
```

Reply:
I like the former, `package.tar.gz`.